pkg/envoy/eds/cluster_load_assignment_test.go

```diff
@@ -25,7 +27,7 @@ func TestNewClusterLoadAssignment(t *testing.T) {
 	}{
 		{
 			name: "multiple endpoints per cluster within the same locality",
-			svc:  service.MeshService{Namespace: "ns1", Name: "bookstore-1", TargetPort: 80},
+			svc:  meshSvc,
```

Comment:
Could you revert this change? The idea of not hardcoding this was to allow new test cases to use a different MeshService and verify the ClusterName corresponds to it.

Reply:
Ok, got it.